azimuth_caas_operator/models/v1alpha1/cluster_type.py

```diff
@@ -26,6 +26,9 @@ class ClusterTypeSpec(schema.BaseModel):
     gitVersion: str
     # Playbook is contained in the above git repo
     playbook: str
+    # The timeout (in seconds) to apply to the kubernetes job resource
+    # which creates, updates and deletes the cluster instances
+    jobTimeout: schema.Optional[int] = pydantic.Field(default=1200)
```

Comment:
Since you are setting a default, this should just be `int` because `None` is not actually a valid value.

Reply:
Really? Will that not force all existing cluster types to be updated? That sounds bad.

Reply:
No, because it will use 1200 if it is not present in the cluster type. That is how default works in this case.

Reply:
The act of having a default is what makes a field optional. `schema.Optional[...]` declares that `None` is a valid value.

Reply:
so you are saying this is optional, and when not there we get 1200 I guess?
`jobTimeout: int = pydantic.Field(default=1200)`

I suspect a few more of these properties are incorrect then!